ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -1029,8 +1030,37 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
                   new ReactTextInputSubmitEditingEvent(
                       editText.getId(), editText.getText().toString()));
 
-              if (blurOnSubmit) {
-                editText.clearFocus();
+              switch (actionId) {
+                case EditorInfo.IME_ACTION_PREVIOUS: {
```

Comment:
This new switch is going to affect both Android phones and TVs.  Is there any chance of regression?